CMakeLists.txt

```diff
@@ -60,10 +60,15 @@ elseif ("${GOOGLE_CLOUD_CPP_GRPC_PROVIDER}" STREQUAL "package")
         # ... use find_package and vcpkg on Windows ...
         find_package(GRPC REQUIRED grpc>=1.4)
         find_package(PROTOBUF REQUIRED protobuf>=3.0)
+        find_path(GTEST_INCLUDE_DIR googletest)
+        find_library(GTEST_LIBRARY googletest)
         link_directories(${GRPC_LIBRARY_DIRS} ${PROTOBUF_LIBRARY_DIRS})
         set(GRPCPP_LIBRARIES gRPC::grpc++)
         set(GRPC_LIBRARIES gRPC::grpc)
         set(PROTOBUF_LIBRARIES protobuf::libprotobuf)
+        # To avoid macro conflict on windows.
+        add_compile_options(/wd4005 /wd4068 /wd4244 /wd4267 /wd4800)
+        add_definitions(/DNOMINMAX /DWIN32_LEAN_AND_MEAN=1 /D_CRT_SECURE_NO_WARNINGS)
```

Comment:
I think these flags are already covered by `cmake/IncludeAbseil.cmake`, please remove the dups.

Reply:
Done.